docs/src/developing/deployed-programs/debugging.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Debugging"
+---
+
+Solana programs run on-chain, so debugging them in the wild can be challenging.
+To make debugging programs easier, developers can write unit tests that directly
+test their program's execution via the Solana runtime, or run a local cluster
+that will allow RPC clients to interact with their program.
+
+## Running unit tests
+
+TODO
+
+## Running on a Local Cluster
```

Comment:
Need help about how we should communicate the best way for folks to run on a local cluster vs unit tests and the next section about how to best do transaction simulation.